### PR TITLE
Convoker ML updates

### PR DESCRIPTION
**Pull request by ghollies**
* ML1 Switched to bounty wood
* ML2 Prescience Node: updated font code to be like storm code,
  but with hp. wired into canDetect logic with new effect
* ML3 power trap: reused logic from decimation trap (using areas)
* ML4 Speed Warp: used font with a new effect type that limits
  speed calculations like a relic does
* ML5 war crystal: kinda just skipped, removed the weight calc.
  Would probably work if we add the mystic_ammo_heat/cold to db.
  need to test
* ML6 Battlewarder: turned into a pulsing focus spell. Icon hangs
  around way too long, but functionally seems to be ok.
  - needed DB update Frequencey 50, Pulse 1
* ML7 DissonanceTrap: reused logic from decimation trap (using areas)
* ML8 BrittleGuard: just changed to ECS effect, with the summon using
  the existing intercept logic
* ML9 SummonMastery: just changed to ECS effect. damage feels pretty
  high, but didnt change the values
* ML10 SummonTitan: Switched to ECS effect, probably needs a DB update
  to make it ground target rather than current posisition. Code
  *should* hopefully support both self/area types on the spell.

### CODE REVIEW

File: GameServer/ECS-Components/AttackComponent.cs [hunk 1]
```diff
@@ -2174,13 +2172,6 @@ public virtual eAttackResult CalculateEnemyAttackResult(WeaponAction action, Att
             if (grapple != null)
                 return eAttackResult.Grappled;
 
-            if (brittleguard != null)
```
Comment (ghollies) [inline GameServer/ECS-Components/AttackComponent.cs:2177]: this is all handled via a 100% intercept rate on the pet. no need for a specific check here

File: GameServer/gameobjects/GamePlayer.cs [hunk 2]
```diff
@@ -12503,8 +12503,10 @@ public virtual bool CanDetect(GamePlayer enemy)
 
             if (this.effectListComponent.ContainsEffectForEffectType(eEffect.TrueSight))
                 return true;
+            if (enemy.effectListComponent.ContainsEffectForEffectType(eEffect.ExposeStealth))
+                return true;
```
Comment (ghollies) [inline GameServer/gameobjects/GamePlayer.cs:12507]: support for ML2, to take effect and make players visible

File: GameServer/propertycalc/MaxSpeedCalculator.cs [hunk 3]
```diff
@@ -111,7 +111,7 @@ public override int CalcValue(GameLiving living, eProperty property)
                         speed *= 2;
                 }
 
-                if (GameRelic.IsPlayerCarryingRelic(player))
+                if (GameRelic.IsPlayerCarryingRelic(player) || player.effectListComponent.ContainsEffectForEffectType(eEffect.SpeedWarp))
```
Comment (ghollies) [inline GameServer/propertycalc/MaxSpeedCalculator.cs:114]: set the max speed, just like relic for speed warp

File: GameServer/spells/SpellHandler.cs [hunk 4]
```diff
@@ -2390,7 +2390,7 @@ public virtual void ApplyEffectOnTarget(GameLiving target)
 				// Harmful spells that deal no damage (ie. debuffs) should still trigger OnAttackedByEnemy.
 				// Exception for DoTs here since the initial landing of the DoT spell reports 0 damage
 				// and the first tick damage is done by the pulsing effect, which takes care of firing OnAttackedByEnemy.
-				if (ad.Damage == 0 && ad.SpellHandler.Spell.SpellType != eSpellType.DamageOverTime)
+				if (ad.Damage == 0 && ad.SpellHandler.Spell.SpellType != eSpellType.DamageOverTime && ad.SpellHandler.Spell.SpellType != eSpellType.Prescience && ad.SpellHandler.Spell.SpellType != eSpellType.SpeedWrap)
```
Comment (ghollies) [inline GameServer/spells/SpellHandler.cs:2393]: making sure speed isnt broken for the caster, and target isnt put in combat when effected by PN/speedwarp

File: GameServer/spells/Valkyrie/PowerRendSpellHandler.cs [hunk 5]
```diff
@@ -60,22 +60,13 @@ public override void OnDirectEffect(GameLiving target)
 
 				if (targetIsGameplayer)
 				{
-					powerRendValue = (int)(target.MaxMana * Spell.Value * GetVariance());
+					powerRendValue = (int)(target.MaxMana * (Spell.Value/100));
```
Comment (ghollies) [inline GameServer/spells/Valkyrie/PowerRendSpellHandler.cs:63]: this works for power trap, where we deal % power damage, but might be a problem for valks. if we need to support them later

File: GameServer/ECS-Services/EffectService.cs [hunk 6]
```diff
@@ -490,6 +492,8 @@ public static eEffect GetEffectFromSpell(Spell spell, bool isBaseLine = true)
                 case eSpellType.SpeedDecrease:
                 case eSpellType.UnbreakableSpeedDecrease:
                     return eEffect.MovementSpeedDebuff;
+                case eSpellType.SpeedWrap:
```
Comment (Fenyn) [inline GameServer/ECS-Services/EffectService.cs:495]:
```suggestion
                case eSpellType.SpeedWarp:
```

**Pull request merged**